src/Offer/CommandHandlers/AddLabelHandler.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Offer\CommandHandlers;
+
+use Broadway\CommandHandling\CommandHandler;
+use CultuurNet\UDB3\Label;
+use CultuurNet\UDB3\Label\LabelServiceInterface;
+use CultuurNet\UDB3\Label\ReadModels\JSON\Repository\ReadRepositoryInterface;
+use CultuurNet\UDB3\Label\ValueObjects\LabelName;
+use CultuurNet\UDB3\Label\ValueObjects\Visibility;
+use CultuurNet\UDB3\Offer\Commands\AddLabel;
+use CultuurNet\UDB3\Offer\OfferRepository;
+use ValueObjects\StringLiteral\StringLiteral;
+
+final class AddLabelHandler implements CommandHandler
+{
+    /**
+     * @var OfferRepository
```

Comment:
This class already proofs its value.

Reply:
Yes this makes it possible to simplify the command handling a lot!

Reply:
I'm going to add a ticket to clean up the other offer commands & handlers like this in the near future

Reply:
I already did the refactoring for these ones because I was already touching the label handlers to make sure the new label aggregates always get created in the handler instead of a service that can be circumvented by accident